cmd/assaf-http/main.go

```diff
@@ -315,6 +325,16 @@ func subscribe(queueName string, sessions chan chan session, consumeRequests cha
 				}
 
 				if consumeRequest.reqId == routingKey {
+					spCtx, _ := amqptracer.Extract(msg.Headers)
```

Comment:
after the round trip has finished we need to match the incoming span to the one we sent so we extract it from the message headers.